platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -1,8 +1,13 @@
 The Gradle team is excited to announce Gradle @version@.
 
 This is the first patch release for Gradle 8.11.
+
 It fixes the following issues:
-* TODO
+
+* [#31268](https://github.com/gradle/gradle/issues/31268) BuildEventsListenerRegistry corrupted with Isolated Projects and parallel configuration
+* [#31278](https://github.com/gradle/gradle/issues/31278) 8.11 Configuration cache failure in settings plugin project.
```

Comment:
:x: #31278 isn't ready yet and may not happen in 8.11.1.
 
But `https://github.com/gradle/gradle/issues/31310` (#31310) is now fixed.